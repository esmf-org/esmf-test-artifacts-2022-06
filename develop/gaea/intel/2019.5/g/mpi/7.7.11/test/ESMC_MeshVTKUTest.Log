build time -- 2022-05-24 01:09:03
20220524 012707.986 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.986 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220524 012707.988 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220524 012707.988 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220524 012707.988 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220524 012707.988 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.988 INFO             PET0 Running with ESMF Version   : v8.3.0b16
20220524 012707.990 INFO             PET0 ESMF library build date/time: "May 24 2022" "01:07:06"
20220524 012707.990 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220524 012707.990 INFO             PET0 ESMF_COMM                   : mpi
20220524 012707.992 INFO             PET0 ESMF_MOAB                   : enabled
20220524 012707.992 INFO             PET0 ESMF_LAPACK                 : enabled
20220524 012707.992 INFO             PET0 ESMF_NETCDF                 : enabled
20220524 012707.992 INFO             PET0 ESMF_PNETCDF                : disabled
20220524 012707.992 INFO             PET0 ESMF_PIO                    : enabled
20220524 012707.992 INFO             PET0 ESMF_YAMLCPP                : enabled
20220524 012707.996 INFO             PET0 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20220524 012707.996 INFO             PET0 Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
20220524 012708.001 INFO             PET0 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20220524 012708.003 INFO             PET0 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20220524 012708.005 INFO             PET0 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20220524 012708.014 INFO             PET0 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20220524 012708.020 INFO             PET0 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20220524 012708.020 INFO             PET0 PASS MeshGetNodeCount, ESMC_MeshVTKUTest.C, line 178
20220524 012708.045 INFO             PET0 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20220524 012708.045 INFO             PET0 PASS MeshGetElementCount, ESMC_MeshVTKUTest.C, line 207
20220524 012708.045 INFO             PET0 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20220524 012708.045 INFO             PET0 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20220524 012708.059 INFO             PET0 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20220524 012708.060 INFO             PET0 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20220524 012708.060 INFO             PET0 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20220524 012708.060 INFO             PET0 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20220524 012708.060 INFO             PET0 Finalizing ESMF
20220524 012707.985 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.986 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220524 012707.988 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220524 012707.988 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220524 012707.988 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220524 012707.988 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.988 INFO             PET1 Running with ESMF Version   : v8.3.0b16
20220524 012707.990 INFO             PET1 ESMF library build date/time: "May 24 2022" "01:07:06"
20220524 012707.990 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220524 012707.990 INFO             PET1 ESMF_COMM                   : mpi
20220524 012707.992 INFO             PET1 ESMF_MOAB                   : enabled
20220524 012707.992 INFO             PET1 ESMF_LAPACK                 : enabled
20220524 012707.992 INFO             PET1 ESMF_NETCDF                 : enabled
20220524 012707.992 INFO             PET1 ESMF_PNETCDF                : disabled
20220524 012707.992 INFO             PET1 ESMF_PIO                    : enabled
20220524 012707.992 INFO             PET1 ESMF_YAMLCPP                : enabled
20220524 012707.996 INFO             PET1 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20220524 012707.996 INFO             PET1 Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
20220524 012708.001 INFO             PET1 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20220524 012708.003 INFO             PET1 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20220524 012708.004 INFO             PET1 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20220524 012708.014 INFO             PET1 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20220524 012708.020 INFO             PET1 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20220524 012708.020 INFO             PET1 PASS MeshGetNodeCount, ESMC_MeshVTKUTest.C, line 178
20220524 012708.045 INFO             PET1 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20220524 012708.045 INFO             PET1 PASS MeshGetElementCount, ESMC_MeshVTKUTest.C, line 207
20220524 012708.045 INFO             PET1 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20220524 012708.045 INFO             PET1 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20220524 012708.062 INFO             PET1 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20220524 012708.063 INFO             PET1 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20220524 012708.063 INFO             PET1 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20220524 012708.063 INFO             PET1 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20220524 012708.063 INFO             PET1 Finalizing ESMF
20220524 012707.979 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.986 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220524 012707.988 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220524 012707.988 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220524 012707.988 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220524 012707.988 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.988 INFO             PET2 Running with ESMF Version   : v8.3.0b16
20220524 012707.990 INFO             PET2 ESMF library build date/time: "May 24 2022" "01:07:06"
20220524 012707.990 INFO             PET2 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220524 012707.990 INFO             PET2 ESMF_COMM                   : mpi
20220524 012707.993 INFO             PET2 ESMF_MOAB                   : enabled
20220524 012707.993 INFO             PET2 ESMF_LAPACK                 : enabled
20220524 012707.993 INFO             PET2 ESMF_NETCDF                 : enabled
20220524 012707.993 INFO             PET2 ESMF_PNETCDF                : disabled
20220524 012707.993 INFO             PET2 ESMF_PIO                    : enabled
20220524 012707.993 INFO             PET2 ESMF_YAMLCPP                : enabled
20220524 012707.996 INFO             PET2 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20220524 012707.996 INFO             PET2 Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
20220524 012708.001 INFO             PET2 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20220524 012708.003 INFO             PET2 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20220524 012708.005 INFO             PET2 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20220524 012708.014 INFO             PET2 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20220524 012708.020 INFO             PET2 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20220524 012708.020 INFO             PET2 PASS MeshGetNodeCount, ESMC_MeshVTKUTest.C, line 178
20220524 012708.045 INFO             PET2 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20220524 012708.045 INFO             PET2 PASS MeshGetElementCount, ESMC_MeshVTKUTest.C, line 207
20220524 012708.045 INFO             PET2 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20220524 012708.045 INFO             PET2 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20220524 012708.094 INFO             PET2 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20220524 012708.094 INFO             PET2 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20220524 012708.094 INFO             PET2 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20220524 012708.095 INFO             PET2 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20220524 012708.095 INFO             PET2 Finalizing ESMF
20220524 012707.985 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.986 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220524 012707.988 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220524 012707.988 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220524 012707.988 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220524 012707.988 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.988 INFO             PET3 Running with ESMF Version   : v8.3.0b16
20220524 012707.990 INFO             PET3 ESMF library build date/time: "May 24 2022" "01:07:06"
20220524 012707.990 INFO             PET3 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220524 012707.990 INFO             PET3 ESMF_COMM                   : mpi
20220524 012707.992 INFO             PET3 ESMF_MOAB                   : enabled
20220524 012707.992 INFO             PET3 ESMF_LAPACK                 : enabled
20220524 012707.992 INFO             PET3 ESMF_NETCDF                 : enabled
20220524 012707.992 INFO             PET3 ESMF_PNETCDF                : disabled
20220524 012707.992 INFO             PET3 ESMF_PIO                    : enabled
20220524 012707.992 INFO             PET3 ESMF_YAMLCPP                : enabled
20220524 012707.996 INFO             PET3 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20220524 012707.996 INFO             PET3 Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
20220524 012708.001 INFO             PET3 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20220524 012708.003 INFO             PET3 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20220524 012708.005 INFO             PET3 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20220524 012708.014 INFO             PET3 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20220524 012708.020 INFO             PET3 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20220524 012708.020 INFO             PET3 PASS MeshGetNodeCount, ESMC_MeshVTKUTest.C, line 178
20220524 012708.045 INFO             PET3 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20220524 012708.045 INFO             PET3 PASS MeshGetElementCount, ESMC_MeshVTKUTest.C, line 207
20220524 012708.045 INFO             PET3 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20220524 012708.045 INFO             PET3 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20220524 012708.062 INFO             PET3 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20220524 012708.062 INFO             PET3 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20220524 012708.062 INFO             PET3 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20220524 012708.062 INFO             PET3 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20220524 012708.062 INFO             PET3 Finalizing ESMF
20220524 012707.981 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.986 INFO             PET4 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220524 012707.988 INFO             PET4 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220524 012707.988 INFO             PET4 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220524 012707.988 INFO             PET4 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220524 012707.988 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.988 INFO             PET4 Running with ESMF Version   : v8.3.0b16
20220524 012707.990 INFO             PET4 ESMF library build date/time: "May 24 2022" "01:07:06"
20220524 012707.990 INFO             PET4 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220524 012707.990 INFO             PET4 ESMF_COMM                   : mpi
20220524 012707.993 INFO             PET4 ESMF_MOAB                   : enabled
20220524 012707.993 INFO             PET4 ESMF_LAPACK                 : enabled
20220524 012707.993 INFO             PET4 ESMF_NETCDF                 : enabled
20220524 012707.993 INFO             PET4 ESMF_PNETCDF                : disabled
20220524 012707.993 INFO             PET4 ESMF_PIO                    : enabled
20220524 012707.993 INFO             PET4 ESMF_YAMLCPP                : enabled
20220524 012707.996 INFO             PET4 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20220524 012707.996 INFO             PET4 Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
20220524 012708.001 INFO             PET4 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20220524 012708.004 INFO             PET4 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20220524 012708.005 INFO             PET4 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20220524 012708.015 INFO             PET4 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20220524 012708.020 INFO             PET4 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20220524 012708.020 INFO             PET4 PASS MeshGetNodeCount, ESMC_MeshVTKUTest.C, line 178
20220524 012708.045 INFO             PET4 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20220524 012708.045 INFO             PET4 PASS MeshGetElementCount, ESMC_MeshVTKUTest.C, line 207
20220524 012708.045 INFO             PET4 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20220524 012708.045 INFO             PET4 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20220524 012708.070 INFO             PET4 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20220524 012708.070 INFO             PET4 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20220524 012708.070 INFO             PET4 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20220524 012708.070 INFO             PET4 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20220524 012708.070 INFO             PET4 Finalizing ESMF
20220524 012707.982 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.986 INFO             PET5 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220524 012707.988 INFO             PET5 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220524 012707.988 INFO             PET5 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220524 012707.988 INFO             PET5 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220524 012707.988 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220524 012707.988 INFO             PET5 Running with ESMF Version   : v8.3.0b16
20220524 012707.990 INFO             PET5 ESMF library build date/time: "May 24 2022" "01:07:06"
20220524 012707.990 INFO             PET5 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220524 012707.990 INFO             PET5 ESMF_COMM                   : mpi
20220524 012707.992 INFO             PET5 ESMF_MOAB                   : enabled
20220524 012707.992 INFO             PET5 ESMF_LAPACK                 : enabled
20220524 012707.992 INFO             PET5 ESMF_NETCDF                 : enabled
20220524 012707.992 INFO             PET5 ESMF_PNETCDF                : disabled
20220524 012707.992 INFO             PET5 ESMF_PIO                    : enabled
20220524 012707.992 INFO             PET5 ESMF_YAMLCPP                : enabled
20220524 012707.996 INFO             PET5 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20220524 012707.996 INFO             PET5 Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
20220524 012708.001 INFO             PET5 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20220524 012708.004 INFO             PET5 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20220524 012708.005 INFO             PET5 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20220524 012708.015 INFO             PET5 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20220524 012708.020 INFO             PET5 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20220524 012708.020 INFO             PET5 PASS MeshGetNodeCount, ESMC_MeshVTKUTest.C, line 178
20220524 012708.045 INFO             PET5 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20220524 012708.045 INFO             PET5 PASS MeshGetElementCount, ESMC_MeshVTKUTest.C, line 207
20220524 012708.045 INFO             PET5 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20220524 012708.045 INFO             PET5 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20220524 012708.069 INFO             PET5 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20220524 012708.069 INFO             PET5 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20220524 012708.069 INFO             PET5 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20220524 012708.069 INFO             PET5 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20220524 012708.069 INFO             PET5 Finalizing ESMF
