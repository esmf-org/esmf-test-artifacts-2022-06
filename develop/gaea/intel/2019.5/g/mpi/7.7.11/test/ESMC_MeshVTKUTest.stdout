build time -- 2022-04-15 00:44:27
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
Beginning Test, file ESMC_MeshVTKUTest.C, line 63NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
Ending Test, file ESMC_MeshVTKUTest.C, line 301 PET 0 Test Elapsed Time -920.212 msec.
Ending Test, file ESMC_MeshVTKUTest.C, line 301 PET 1 Test Elapsed Time -914.64 msec.
Ending Test, file ESMC_MeshVTKUTest.C, line 301 PET 2 Test Elapsed Time -922.78 msec.
Ending Test, file ESMC_MeshVTKUTest.C, line 301 PET 3 Test Elapsed Time -900.389 msec.
Ending Test, file ESMC_MeshVTKUTest.C, line 301 PET 4 Test Elapsed Time -921.878 msec.
Ending Test, file ESMC_MeshVTKUTest.C, line 301 PET 5 Test Elapsed Time -921.008 msec.
